app/forms/funding/form_validator.rb

```diff
@@ -12,7 +12,8 @@ class FormValidator
 
     validate :validate_training_initiative
 
-    delegate :applying_for_bursary, to: :bursary_form
+    delegate :applying_for_bursary, :applying_for_scholarship,
```

Comment:
Not used here I don't think.

```suggestion
    delegate :applying_for_bursary,
```

Reply:
Actually it is needed, it's used in the component

Reply:
ma bad!